From 4eecec190132fad3cff2ac81dcaf02d48e95feab Mon Sep 17 00:00:00 2001
From: Quoc Ngo <quoc.ngo.wz@renesas.com>
Date: Thu, 23 May 2019 17:29:05 +0700
Subject: [PATCH 86/92] Revert "arm64:defconfig: add REALTEK_PHY for the
 ethernet"

Currently, Realtek phy has a link up/down issue when using ifconfig
command. To avoid this issue, we disable Realtek phy and use Generic phy
as a workaround

This reverts commit 2997d51ae2482480d16231d3a4941326f83da451.

Signed-off-by: Quoc Ngo <quoc.ngo.wz@renesas.com>
---
 arch/arm64/configs/defconfig | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm64/configs/defconfig b/arch/arm64/configs/defconfig
index 9be1769..099d1ac 100644
--- a/arch/arm64/configs/defconfig
+++ b/arch/arm64/configs/defconfig
@@ -244,7 +244,7 @@ CONFIG_MARVELL_PHY=m
 CONFIG_MARVELL_10G_PHY=m
 CONFIG_MESON_GXL_PHY=m
 CONFIG_MICREL_PHY=y
-CONFIG_REALTEK_PHY=y
+CONFIG_REALTEK_PHY=m
 CONFIG_ROCKCHIP_PHY=y
 CONFIG_USB_PEGASUS=m
 CONFIG_USB_RTL8150=m
-- 
2.7.4

